skipper-mixin/README.md

```diff
@@ -0,0 +1,23 @@
+# Skipper Mixin
+
+The Skipper Mixin is a set of configurable, reusable, and extensible alerts and
+dashboards based on the metrics exported by the MySQLd Exporter. The mixin creates
+recording and alerting rules for Prometheus and suitable dashboard descriptions
+for Grafana.
+
+To use them, you need to have `mixtool` and `jsonnetfmt` installed. If you
```

Comment:
aren't mixtool and jsonnetfmt required only for building the mixin?

Reply:
Yes because the dashboard is in json at the moment, we only need `mixtools` and `jsonnetfmt` for building the mixin. Will clarify in the README.